.github/config.yml

```diff
@@ -0,0 +1,9 @@
+# ProBot TODO bot
+# https://probot.github.io/apps/todo/
+
+todo:
+  autoAssign: false
+  blobLines: 7
+  caseSensitive: true
+  keyword: "TODO"
```

Comment:
we also have `fixme` and `xxx` in our code, some are in lowercase.

Reply:
This will work only for new code, anyway. The bot does not accept multiple keywords as far as I know and `TODO` is the standard way.